### ISSUE TITLE
Heap Buffer

### ISSUE THREAD

**Issue by RwNigma** [DarkstarProject/darkstar#2284]
After pulling a large amount of enemies within The Boyahda Tree there is a possibility of this error: 

"HEAP CORRUPTION DETECTED: after Normal Block (#10106553) at 0x265CCA70 CRT detected that the application wrote to memory after end of heap buffer."

This seemed to happen not when the mobs were moving, but when they were all going to attack all at once.

**Comment by teschnei** [comment]
Can't reproduce.  Are you sure it was not #2219?

**Comment by RwNigma** [comment]
I was using my own Test Server, so it was only me on the server. I was able to have it happen 3 times before I posted. 

I started from the Knight Crawlers within the small tunnel. I ran around and grabbed mandy's, rabbits, robber crabs and Goobbue's. I didn't stop until I got to where Unut spawns. The mobs ran to me and once they were all around me and had hit me once or twice I got the error message and it crashed. I can count the mobs to see exactly how many it was.
